db/crud.go

```diff
@@ -2582,3 +2585,25 @@ func (db *DatabaseCollectionWithUser) CheckProposedRev(ctx context.Context, doci
 		return ProposedRev_Conflict, doc.CurrentRev
 	}
 }
+
+const (
+	xattrMacroCas         = "cas"
```

Comment:
should we pull this value from sg-bucket? not convinced either way

Reply:
I think it belongs in SGW as this defines the document property that's using the macro expansion, not the macro expansion type.